web/themes/custom/novel/templates/dpl-react-app--favorites-list.html.twig

```diff
@@ -1,6 +1,7 @@
 <div class="dpl-react-app-container--page">
     <div {{ attributes }}>
         <div class="card-list-page">
+            {# Todo: use content-list-page__heading #}
```

Comment:
TODO?

Reply:
They are here to help when we start working on https://reload.atlassian.net/browse/DDFFORM-953

Reply:
Ah, okay. In general I'm not as opposed to todos as others, but if there's already an issue it would be nice mentioning it in the comment, as it gives anyone stumbling across it more context.